e_separate_NW_CLP_data/src/subset_file_by_data_group.R

```diff
@@ -0,0 +1,41 @@
+#' @title subset the RS summary data to those belonging to specific ROSS projects
+#' 
+#' @description
+#' This function splits data into specified data groups, as defined in group -a-
+#' 
+#' @param data_file .feather filepath; remote sensing summary data, specifically those that
+#' have had the handoff coefficient applied. 
+#' @param data_grp character string; in this case, "NW" or "CLP", but could be any
+#' data group specified in group -a-
+#' 
+#' @returns filepath of resulting .feather file
+#' 
+#' 
+subset_file_by_data_group <- function(data_file, data_grp) {
+  data <- read_feather(data_file)
+  subset <- data %>% 
+    filter(data_group == !!data_grp)
```

Comment:
I've oddly never seen this before. what is the benefit of the `!!` ?

Reply:
Apparently it's called an 'injector' - and often works similarly to `{{ data_grp }}`, but not always. I'll be honest that I usually just try {{ var }} and !! var as necessary to get the expected output from a function... It also has something to do with the difference between 'symbolics' and 'quosures' which I truly don't understand. 

This is, by all means, and absolutely terrible non-explanation of your perfectly good question. If I manage to get a better grasp, I'll most certainly let you know. :)